CosmicReco/src/SimpleTimeCluster_module.cc

```diff
@@ -160,16 +167,20 @@ void SimpleTimeCluster::findClusters(TimeClusterCollection& tccol) {
       count += ordChCol[endIndex].nStrawHits();
       if (endIndex >= ordChCol.size())
         break;
-      if (endTime - ordChCol[endIndex - 1].correctedTime() > _timeStep)
+      if (_usetimeStep && endTime - ordChCol[endIndex - 1].correctedTime() > _timeStep)
         break;
       if (endTime - startTime < 0)
         break;
+      if (_usetimeWindow && endTime - startTime > _timeWindow)
+        break;
     }
     count -= ordChCol[endIndex].nStrawHits();
     endIndex--;
     endTime = ordChCol[endIndex].correctedTime();
     if (_hasmaxnsh && count > _maxnsh)
       continue;
+    if (!_usetimeWindow && endTime - startTime > 100)
```

Comment:
I think 100 here is supposed to be _timeWindow

Reply:
Yes, thanks! Fixed